client/src/components/Modal.jsx

```diff
@@ -1,15 +1,24 @@
 import React from 'react';
+import Review from './Review.jsx';
+import ReviewContainer from './ReviewContainer.jsx';
 import styles from './css/modalStyles.css.js';
 
 const Modal = (props) => {
   if (!props.showModal) {
     return null;
   } else {
     return (
-      <div style={styles.modal}>
+      <div style={styles.modal} onClick={props.onClose}>
         <section style={styles.modalMain}>
-          <img src="https://i.etsystatic.com/iap/c08960/1735567682/iap_640x640.1735567682_9rntz5h8.jpg?version=0"></img>
-          <button onClick={props.onClose}>Close</button>
+          {/* <div style={styles.reviewContent}> */}
+          <img style={styles.image} src="https://i.etsystatic.com/iap/c08960/1735567682/iap_640x640.1735567682_9rntz5h8.jpg?version=0"></img>
```

Comment:
I highly recommend you don't link to etsy's images.  save them locally and serve them from your static files/S3

Reply:
I started with hard coding some data to get some styling going. I starting to work on getting my backend hooked up now